### ISSUE TITLE
WIndows List Scrolling

### ISSUE THREAD

**Issue by DarthGandalf** [kvirc/KVIrc#1301]
**Reported by Cronus on 05 Jul 2012 01:31:04 UTC**
Pertaining to the setting boolWheelScrollsWindowsList Which either when using the mouse wheel you scroll the active window list, or the scrollbar is the one scrolling.  If you do these specific steps it will spazz out and try to scroll the active window and scrollbar at the same time..

I am running windows 7 64bit FYI

Minimize KVIRC
Go to another application (by clicking on it and bringing it into the active window
Now Click on kvirc to maximize it again and try to use your mouse wheel in the windows list on the left.

---

Migrated from: https://svn.kvirc.de/kvirc/ticket/1301

**Comment by DarthGandalf** [comment]
**Comment by ctrlaltca on 16 Jul 2012 21:45:34**
You forgot to mention the actual value of the boolWheelScrollsWindowsList option you're using. Anyway i tried to perform the steps you described with both that option turned on or off:
- when it's off, only the scrollbar scrolls;
- when it's on, the active window in the window list changes; when the last visible item in the window list gets selected, but there are some others more after it, the scrollbar scrolls to make the very next item visible so that it can gets selected and become the active window. This is the intended behaviour to ensure that the currently selected window list item is visible.<br/>

Is this what you mean or did i miss the point?

**Comment by DarthGandalf** [comment]
**Comment by Cronus on 16 Jul 2012 23:04:00**
I have it turned off. And no that is not I mean. But like I said it basically tries to do BOTH actions at the same time (Change the active window AND move the scrollbar) I showed someone from the freenode support channel. I can't remember who it was, it might have been you? I showed them over join.me session and they said to make this ticket and maybe someone on windows can assist.

**Comment by un1versal** [comment]
cannot duplicate this currently, nothing unusual about this as is, imo this unlikely scenario only a edge case user sees and also its on a outdated buggy build.

**Comment by un1versal** [comment]
I can actually dup this after a couple of tries... its odd after doing this a channel is both selected and you can have some scrolling selection at same time. looks and behaves oddly and looks like arse too.

But like everything, this in particular is a very edge case scenario you wouldn't normally see or experience unless trying to find this issue.

**Comment by un1versal** [comment]
> Arfrever commented on 3 Mar 

If I press long "rectangle" used to scroll list of channels and move cursor up or down AND to the left so that it points to name of any channel and if I stop pressing mouse button in this moment, then name of that channel will become semi-highlighted.

(Semi-highlighting does not stop when moving cursor over name of that channel.)

#fcitx in below example:
![kvirc_channels](https://cloud.githubusercontent.com/assets/656294/13501569/8474c854-e167-11e5-88b6-573df79dd3b4.png)

```
Executable Information

KVIrc 4.9.1 'Aria'

Runtime Info:
System name: Linux
Architecture: x86_64
Qt version: 5.5.1
Qt theme: fusion

Build Info:
Build date: 2016-02-25 03:59:53 UTC
Sources date: 20160102
Revision number: fb3d7d44b0a90afea71c1775d4511e3807ab2c63
System name: Linux
CPU name: x86_64
Build command: /usr/bin/cmake
Build flags: 
   MANDIR=share/man
   CMAKE_INSTALL_PREFIX=/usr
   VERBOSE
   LIB_SUFFIX=64
   Threads=POSIX
Compiler name: /usr/bin/x86_64-pc-linux-gnu-g++
Compiler flags: -march=native -O2 -fno-ident -frecord-gcc-switches -pipe -Wall -fno-strict-aliasing 
Qt version: 5.5.1
```

---

Merged in #1897 on **Wed Mar  9 12:52:51 GMT 2016**

**Comment by un1versal** [comment]
I can confirm this issue when following duplication steps in #1940 also
